.pre-commit-config.yaml

```diff
@@ -1,5 +1,6 @@
 ci:
   autoupdate_schedule: monthly
+  skip: ["check-stack-tag-values"] # work around for https://github.com/pre-commit-ci/issues/issues/241
```

Comment:
I think it's important to run this check on PRs, but since we can't run it on the pre-commit workers can you also add a github action that runs `pre-commit run check-stack-tag-values`?

Reply:
so, the idea is to not run this check on pre-commit.ci however [it is setup to run with github action runners](https://github.com/Sage-Bionetworks-IT/aws-ami-monorepo/blob/main/.github/workflows/deploy.yaml#L14), so we should be covered.  the downside is that the workflow will run a little slower because we run the other checks both with pre-commit.ci and GH runner.